### PR TITLE
fix: add fs.stat err guard

### PR DESCRIPTION
**Pull request by markwolff**
- Adds error handler for where file is inaccessible due to file permissions or being deleted by another process. Some environments might also reset the tmp directory during restart.

### CODE REVIEW

File: examples/grpc/package.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
     "build": "echo 'nothing to build'",
     "test": "echo 'nothing to test'",
     "lint": "echo 'nothing to lint'",
-    "server": "cross-env APPINSIGHTS_INSTRUMENTATIONKEY=ikey node ./server.js",
-    "client": "cross-env APPINSIGHTS_INSTRUMENTATIONKEY=ikey node ./client.js"
+    "server": "cross-env APPINSIGHTS_INSTRUMENTATIONKEY=3457abac-3812-4174-a542-e21d8d09c88f node ./server.js",
+    "client": "cross-env APPINSIGHTS_INSTRUMENTATIONKEY=3457abac-3812-4174-a542-e21d8d09c88f node ./client.js"
```
Comment (xiao-lix) [inline examples/grpc/package.json]: Are these ikeys exposed on purpose?

File: packages/azure-opentelemetry-exporter/.eslintrc.js [hunk 2]
```diff
@@ -16,9 +16,11 @@ module.exports = {
...
     'node/no-unsupported-features/es-syntax': ['error', { ignores: ['modules'] }],
     'import/prefer-default-export': 'off',
-    "import/no-extraneous-dependencies": ["error", {"devDependencies": true}],
-    "node/no-unpublished-import": ["error", {
-        "allowModules": ["@opentelemetry/api", "@opentelemetry/tracing"]
+    "import/no-extraneous-dependencies": ["error", {
+      "devDependencies": ["**/*.test.ts"]
+    }],
+    "node/no-extraneous-import": ["error", {
+      "allowModules": ["nock"]
```
Comment (xiao-lix) [inline packages/azure-opentelemetry-exporter/.eslintrc.js]: nit: should it use single quotes here consistent with other fields?

**Review by xiao-lix — APPROVED** [review]

File: packages/azure-opentelemetry-exporter/.eslintrc.js (hunk 2 above)
Comment (markwolff) [reply packages/azure-opentelemetry-exporter/.eslintrc.js]: :+1: switched to single quotes

File: examples/grpc/package.json (hunk 1 above)
Comment (markwolff) [reply examples/grpc/package.json]: nope. removed them

**Pull request merged**